Task72/Task72/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task72
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите x и y");
+
+            int x = int.Parse(Console.ReadLine());
+            int y = int.Parse(Console.ReadLine());
```

Comment:
Выбран неверный тип данных. Координаты точки на плоскости **не обязательно целые числа**.
```suggestion
            double x = double.Parse(Console.ReadLine());
            double y = double.Parse(Console.ReadLine());
```